staging/src/k8s.io/code-generator/cmd/client-gen/args/args.go

```diff
@@ -27,12 +27,7 @@ import (
 	codegenutil "k8s.io/code-generator/pkg/util"
 )
 
-var DefaultInputDirs = []string{
-	"k8s.io/apimachinery/pkg/fields",
```

Comment:
these were introduced in https://github.com/kubernetes/kubernetes/pull/17534/files#diff-c355f7874eb79825bea5290fd48a018aR35. If I am not mistaken, we auto-import packages nowadays if they are referenced within generated code (via import tracker). Maybe this was different in the beginning. @caesarxuchao can you comment? Removing these dirs here does not change the generator's output.